### PR TITLE
[time-span] introduce typings

### PR DESCRIPTION
**Pull request by BendingBender**
Definitions inspired by https://github.com/sindresorhus/time-span/pull/1.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

File: types/time-span/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// Type definitions for time-span 2.0
+// Project: https://github.com/sindresorhus/time-span#readme
+// Definitions by: BendingBender <https://github.com/BendingBender>
+//                 Mike Dvorscak <https://github.com/mdvorscak>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace timeSpan {
+    type end = () => number;
```
Comment (weswigham) [inline types/time-span/index.d.ts]: Rather than having this type (at all),

File: types/time-span/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+    interface TimeSpanObject {
+        rounded(): number;
```
Comment (weswigham) [inline types/time-span/index.d.ts]: Just add `(): number;` to this interface.

File: types/time-span/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        sec(): number;
+        nano(): number;
+    }
+}
+
+declare function timeSpan(): timeSpan.end & timeSpan.TimeSpanObject;
```
Comment (weswigham) [inline types/time-span/index.d.ts]: And remove `timeSpan.end & `.

**Review by weswigham — CHANGES_REQUESTED** [review]
Just a request for a small simplification.

**Comment by BendingBender** [comment]
@weswigham thank you for your review!

**Pull request merged**